py/estimator_runconfig_tests.py

```diff
@@ -99,24 +108,34 @@ def test_tfjob_and_verify_runconfig(self):
     # Wait for the job to either be in Running state or a terminal state
     logging.info("Wait for conditions Running, Succeeded, or Failed")
     results = tf_job_client.wait_for_condition(
-      api_client, self.namespace, self.name, ["Running", "Succeeded", "Failed"],
-      version=self.tfjob_version, status_callback=tf_job_client.log_status)
+      api_client,
+      self.namespace,
+      self.name, ["Running", "Succeeded", "Failed"],
+      version=self.tfjob_version,
+      status_callback=tf_job_client.log_status)
     logging.info("Current TFJob:\n %s", json.dumps(results, indent=2))
 
-    num_ps = results.get("spec", {}).get("tfReplicaSpecs", {}).get(
-      "PS", {}).get("replicas", 0)
+    num_ps = results.get("spec", {}).get("tfReplicaSpecs",
+                                         {}).get("PS", {}).get("replicas", 0)
```

Comment:
Are this changes from autopep8 or some similar tools?

Reply:
I follow the developer_guide  and use 
```
yapf -i py/**/*.py
```
to lint the python code